pages/404.tsx

```diff
@@ -0,0 +1,42 @@
+import Wrapper from "@components/SDCWrapper";
+import SvgErrorIcon from "@components/icons/ErrorIcon";
+import { Button, Box, Center, Heading, Text } from "@chakra-ui/react";
+
+type ComponentProps = {
+    session: Record<string, unknown>;
+};
+
+export default function Custom404(props: ComponentProps): JSX.Element {
+    const goToHome = () => {
+        window.location.href = "/";
+    };
+
+    return (
+        <Wrapper session={props.session}>
+            <Center minHeight="85vh" align="center">
+                <Box align="center" width="70%">
+                    <SvgErrorIcon />
+                    <Heading size="md" margin="3% 0% 3% 0%">
+                        Oh no! Page not found.
+                    </Heading>
+                    <Text size="md" marginBottom="3%">
+                        Sorry, but the page you are looking for does not exist.
+                        Try refreshing the page or hit the button below.
+                    </Text>
+                    <Button
+                        color="white"
+                        backgroundColor="#0C53A0"
```

Comment:
Please `import colourTheme from "@styles/colours";` and get colours from a source of truth
```suggestion
                        backgroundColor={colourTheme.colors.Blue}
```